gitpod-io/website
# Updated description of self hosted cluster guides

## Description
- Updated each page that describes a cloud cluster creation guide to be in line with the new intended use of the guides: to create a cluster, not create a cluster AND install Gitpod

## How to test
Spin up workspace, open preview on port 3000, navigate to Docs > Self-Hosted> Cluster Set-Up. Then you can click on each of the guides to see the respective pages.

## Release Notes

```release-note
NONE
```


<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1971"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1971"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
